### PR TITLE
Feature/2019 initial

### PR DESCRIPTION
**Pull request by alexsavio**
This PR adds a few known details about the conference. Also Privacy and CoC.
I mainly need you to check the conference chairs.

### CODE REVIEW

File: content/pages/2019/about.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Oier Etxaniz (ACPySS)
+- Alexandre Manhaes Savio (ACPySS)
+
+#### Program chair
+
+- Alexander CS Hendorf (Königsweg GmbH, DE)
+
+#### Tutorials chair
+
+- Emmanuelle Gouillart (Joint Unit CNRS/Saint-Gobain, FR)
```
Comment (landeraxe) [inline content/pages/2019/about.md]: Emma is not available this year, it's Giovanni De Gasperis, Ph.D  (Computer Science and Mathematics
University of L'Aquila)

Comment (alanderex) [inline content/pages/2019/about.md]: Emma is not available this year, it's Giovanni De Gasperis, Ph.D  (Computer Science and Mathematics
University of L'Aquila)

File: content/pages/2019/about.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+#### Organizing committee
+
+- Pierre de Buyl (KU Leuven, BE)
+- Mike Müller (Python Academy, DE)
+- Alessia Marcolini (Fondazione Bruno Kessler - FBK)
+- Susanne Greiner
+
```
Comment (alanderex) [inline content/pages/2019/about.md]: - Marc Garcia
- Oswald ?

File: content/pages/2019/about.md (hunk 1 above)
Comment (giodegas) [reply content/pages/2019/about.md]: to be precise, my affiliation is "Department of Information Engineering, Computer Science and Mathematics, University of L'Aquila, Italy" http://www.disim.univaq.it

File: content/pages/2019/about.md (hunk 2 above)
Comment (alexsavio) [reply content/pages/2019/about.md]: done. thx

File: content/pages/2019/about.md (hunk 1 above)
Comment (alexsavio) [reply content/pages/2019/about.md]: done. thx

Comment (alexsavio) [reply content/pages/2019/about.md]: do you mind if I leave just the University name and country, as the others?

**Comment by pdebuyl** [comment]
@alexsavio let me know when I can merge and publish.

File: content/pages/2019/about.md (hunk 1 above)
Comment (giodegas) [reply content/pages/2019/about.md]: no problem, make it shorter as needed. Just to give you the full record from were you can summarize at will. Thank's. :+1:

**Comment by alexsavio** [comment]
Thanks @pdebuyl. Good to be merged, for now. Will do modifications if needed.

**Comment by pdebuyl** [comment]
Are the contact persons for the CoC up to date for 2019?

**Comment by alexsavio** [comment]
yes. I am open to any other volunteer though

**Comment by pdebuyl** [comment]
Merged and live. If you look for people, you can start with the org mailing list.

**Pull request merged**